mono/mini/mini-arm64.h

```diff
@@ -177,9 +177,6 @@ typedef struct {
 #define MONO_ARCH_FLOAT32_SUPPORTED 1
 #define MONO_ARCH_HAVE_INTERP_PINVOKE_TRAMP 1
 #define MONO_ARCH_LLVM_TARGET_LAYOUT "e-i64:64-i128:128-n32:64-S128"
-#ifdef TARGET_OSX
```

Comment:
Nit, I always pause at deleting upstream code. Is there some way we want to capture that we removed this?

Reply:
Good point, I'll do it with another define.